jaeger-core/src/test/java/com/uber/jaeger/samplers/GuaranteedThroughputSamplerTest.java

```diff
@@ -48,8 +50,8 @@ public void testRateLimitingLowerBound() throws Exception {
     Assert.assertTrue(samplingStatus.isSampled());
     Map<String, Object> tags = samplingStatus.getTags();
 
-    assertEquals(tags.get(Constants.SAMPLER_TYPE_TAG_KEY), RateLimitingSampler.TYPE);
-    assertEquals(tags.get(Constants.SAMPLER_PARAM_TAG_KEY), 1.0);
+    assertEquals(tags.get(Constants.SAMPLER_TYPE_TAG_KEY), GuaranteedThroughputSampler.TYPE);
+    assertEquals(tags.get(Constants.SAMPLER_PARAM_TAG_KEY), 0.0001);
```

Comment:
It's an artifact for when I first designed this. The lowerbound sampler needed to return the probability that **didn't** sample it so on the backend we know that we should bump this probability. However, given that the backend is the source of probability now, this isn't needed but I'm keeping it for consistency sake before I rip it out of all the clients and return adaptive sampling tags.